curator/static/js/curation-helpers.js

```diff
@@ -1334,7 +1334,14 @@ function promptForSaveCollectionComments( collection ) {
         $('#save-collection-comment-first-line').val(firstLine);
         $('#save-collection-comment-more-lines').val(moreLines);
 
-        $('#save-collection-comments-popup').modal('show');
+        if (userEmail != 'ANONYMOUS') {
+          // console.log('email: '+ userEmail);
+          $('#save-collection-comments-popup').modal('show');
+        }
+        else {
+          $('#save-collection-comments-popup-no-email').modal('show');
+        }
```

Comment:
Due to the presence of (presumably) unique IDs within this popup, it's probably safer to always use the original popup element and hide/show a warning element inside it. Otherwise on submission we're liable to lose the comment text in one case or the other.